brightlayer-ui/react-workflows
# Feature/2857 85 more efficient import syntax

Fixes #85 


#### Changes proposed in this Pull Request:
- Changed import statements of material-ui components to default imports
- Comp props and other things than components still be imported as named imports

### Screenshots:
**Before**
<img width="1777" alt="Screenshot 2022-01-21 at 4 40 29 PM" src="https://user-images.githubusercontent.com/25982779/150520666-36a0087e-d5e2-427f-962c-443e263b42d5.png">

**After**
<img width="1777" alt="Screenshot 2022-01-21 at 4 54 13 PM" src="https://user-images.githubusercontent.com/25982779/150520706-32be8b2a-12be-4989-b61e-1a4547e1bd90.png">


#### To Test:
1. Start a new React project with the pxblue cli via command line prompt npx -p @brightlayer-ui/cli blui new
2. Run yarn build from the login-workflow of the react-workflows in the updated branch you have locally.
3. The previous command should create a dist folder... copy the contents of that dist folder into your new cli generated projects node_modules/@brightlayer-ui/react-auth-workflow
4. Add the source-map-explorer to the cli generated project as a dev dependency and add "analyze" script "analyze": "source-map-explorer 'build/static/js/*.js'" 
5. Run yarn build in the new cli project
6. Run yarn analyze
7. Compare the new chart to the one posted in the PR
